### PR TITLE
Update yaml for US Sen Mazie Hirono (VAN-50069)

### PR DESCRIPTION
**Pull request by OdeclasV**
I updated this yaml to account for a change in the link to the form and the CSS selectors. In addition, added  a final wait of 2 seconds since the Thank You message took some time to load and yamltron was returning an incomplete screenshot.

### CODE REVIEW

File: members/H001042.yaml [hunk 1]
```diff
@@ -3,131 +3,73 @@ contact_form:
...
         required: true
-      - name: mailing_streetAddress1
-        selector: "#mailing_streetAddress1"
+      - name: street
+        selector: input#input-B03E05F1-4040-F985-52CD-A580E72A7D41
         value: $ADDRESS_STREET
         required: true
-      - name: mailing_streetAddress2
-        selector: "#mailing_streetAddress2"
+      - name: street
```
Comment (tomdionesotes) [inline members/H001042.yaml]: might be helpful to name this street2 for clarity

File: members/H001042.yaml [hunk 2]
```diff
@@ -3,131 +3,73 @@ contact_form:
...
-          "Airman": "Airman"
-      - name: subject
-        selector: "#subject"
+          "Dean": "Dean"
+          "The Honorable": "The Honorable"
+          "Dr.": "Dr."
+          "None": "None"
+      - name: topic
+        selector: select#input-B03E04DB-4040-F985-52CD-18F332336F99
         value: $TOPIC
```
Comment (tomdionesotes) [inline members/H001042.yaml]: do the new topics map to anything in artemis?

Comment (OdeclasV) [reply members/H001042.yaml]: good call. I'll update Artemis with the topics, right now they are all pointing to "Congressional Procedures"

File: members/H001042.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/H001042.yaml]: done!

**Comment by tomdionesotes** [comment]
LGTM!

**Pull request merged**